internal/component/loki/source/aws_firehose/internal/handler_test.go

```diff
@@ -468,3 +468,277 @@ func keepLabelRule(src, dst string) *relabel.Config {
 		Action:       relabel.Replace,
 	}
 }
+
+func TestHandlerWithStaticConfigsLabels(t *testing.T) {
+	type testcase struct {
+		// TenantID configures the X-Scope-OrgID header in the test request when present.
+		TenantID string
+
+		// Body is the payload of the request.
+		Body string
+
+		// Assert is the main assertion function ran after the request is successful.
+		Assert func(t *testing.T, res *httptest.ResponseRecorder, entries []loki.Entry)
+
+		// AssertMetrics is an optional assertion over the collected metrics
+		AssertMetrics      func(t *testing.T, m []*dto.MetricFamily)
+		StaticLabelsConfig string
+	}
+
+	tests := map[string]testcase{
+		"direct put data, static labels": {
+			Body: readTestData(t, "testdata/direct_put.json"),
```

Comment:
Do we need to commit this file?

Reply:
@tpaschalis, I think this file was already committed.

https://github.com/grafana/alloy/blob/main/internal/component/loki/source/aws_firehose/internal/testdata/direct_put.json

Reply:
Ah, apologies then.

Reply:
No worries :)